README.md

```diff
@@ -84,6 +76,14 @@ The related documentation and instructions are [here](schema-registry-examples).
 
 The related documentation and instructions are [here](spark-connector-examples).
 
+## Hadoop Connector Examples (_Retired: available up to Pravega Samples release 0.10_)
+| Example Name  | Description  | Language |
+| ------------- |:-----| :-----|
+| `wordcount` | Counts the words from a Pravega `Stream` filled with random text to demonstrate the usage of Hadoop connector for Pravega. | [Java](hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount)
```

Comment:
Should these become fully-qualified URLs instead of relative paths? These should link to a branch or tag now that they're deleted?

Reply:
Done.